src/utils.js

```diff
@@ -825,3 +825,13 @@ export function rangeSpec(schema) {
   }
   return spec;
 }
+
+export function every(items, predicate) {
```

Comment:
I would prefer to not add every.
There is `.every` on arrays since a long time (pre ES5).

Reply:
Sure thing, I can't believe I missed this!